core/store/tx_manager.go

```diff
@@ -750,23 +783,27 @@ func (txm *EthTxManager) bumpGas(tx *models.Tx, attemptIndex int, blockHeight ui
 	bumpedGasPrice := txm.BumpGasByIncrement(originalGasPrice)
 
 	for {
+		promNumGasBumps.WithLabelValues(tx.Hash.String(), string(attemptIndex))
```

Comment:
Do I need an `Observe` or `Set` here? @jleeh @HenryNguyen5

Reply:
With counters @samsondav it's `Inc` or `Dec`.